site3/website/docs/reference/config.md

```diff
@@ -182,7 +182,8 @@ The table below lists parameters that you can set to configure bookies. All conf
 | compactionRateByEntries | Set the rate at which compaction will read entries. The unit is adds per second. | 1000 | 
 | compactionRateByBytes | Set the rate at which compaction will read entries. The unit is bytes added per second. | 1000000 | 
 | useTransactionalCompaction | Flag to enable/disable transactional compaction. If it is set to true, it will use transactional compaction, which uses<br />new entry log files to store entries after compaction; otherwise, it will use normal compaction, which shares same entry<br />log file with normal add operations.<br /> | false | 
-
+| minorCompactionMaxTimeMillis | Maximum milliseconds to run minor Compaction. | -1 to run indefinitely. | 
```

Comment:
Thanks for your contribution. Maybe put in after `majorCompactInterval` `minorCompactionInterval` is better?

Reply:
good idea, updated.